libs/api/domains/air-discount-scheme/src/lib/api-domains-air-discount-scheme.resolver.spec.ts

```diff
@@ -23,14 +25,22 @@ describe('ApiDomains: AirDiscountSchemeResolver', () => {
     scope: [ApiScope.internal],
   })
 
-  const fabGetDiscount = (nationalId: string): TDiscount => ({
+  const fabGetDiscount = (nationalId: string): DiscountWithTUser => ({
+    user: {
+      ...fabTUser(nationalId),
+      name: '',
```

Comment:
Sure. It's not explicitly nameless, it just needs to be set.
I chose to simply fabricate TUser but the discount model has two sources for it user property, name coming from the User model and thus explicitly set in this test case so that the type system doesn't complain.

I suppose this is clearer with a name so I included it.
:+1: